readme.md

```diff
@@ -562,29 +562,29 @@ $faker = Faker\Factory::create();
 <?xml version="1.0" encoding="UTF-8"?>
 <contacts>
 <?php for ($i=0; $i < 10; $i++): ?>
-  <contact firstName="<?php echo $faker->firstName ?>" lastName="<?php echo $faker->lastName ?>" email="<?php echo $faker->email ?>">
-    <phone number="<?php echo $faker->phoneNumber ?>"/>
+  <contact firstName="<?php echo $faker->firstName; ?>" lastName="<?php echo $faker->lastName; ?>" email="<?php echo $faker->email; ?>">
```

Comment:
no, semicolons are not required in this case, and they make the code less readable